vmdk_plugin/plugin.go

```diff
@@ -209,6 +221,10 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 
+	// Lock this volume for create
+	d.lockVolume(r.Name)
+	defer d.unlockVolume(r.Name)
```

Comment:
Do you know which is the behaviour ifa container starts with a vol that is being removed? Mount() in parallel with Remove() ? Does the docker daemon protects from this?